### ISSUE TITLE
Behavior of filters with center/offset property on moving objects

### ISSUE THREAD

**Issue by 4ian** [pixijs/filters#304]
Hi!

I have some questions about what seems to be inconsistent behavior between a few filters - though I can't be sure if it's working as intended or not.
See this jsfiddle: https://jsfiddle.net/a17oxf92/ - this should be using latest PixiJS and latest filter versions.

- The first filter "Twist" is not following at all the Sprite that is moving on the screen. This means that the "center of the twist" is always at the specified position relative to the screen. Intuitively, the center would be in the local object coordinates and would move along with the object (so you would not see any change to the rendered object, it would always be "twisted in the same way", just moving horizontally). I see that the property for this filter is called `offset` and not `center` so this may be on purpose?
- The second filter "Radial blur" is properly following the Sprite that is moving, but up to a point. 
  - If the sprite goes to the right, it's not changing. If it's going "too much" to the right, it's starting to change a bit (see the black areas appearing on the bottom right):
    ![image](https://user-images.githubusercontent.com/1280130/129764652-eb32f07b-01fd-4054-9193-d33fb4359ad2.png)

  - If the sprite goes to the left, the center is now staying fixed on the screen, not following the Sprite anymore (i.e: the radial blur is not centered anymore): 
    ![image](https://user-images.githubusercontent.com/1280130/129764570-f11247e0-296a-4f92-8fd6-70c0ca15ad5a.png)
- The third filter "Zoom blur" is "working well" when going to the right (i.e: the Sprite is not changing) but when going too much to the left, it's starting to "drift" (a bit like the radial blur).

It would be helpful to understand if it's a misunderstanding from my side on these 3 filters. The "drifting" center of radial/zoom when going too much on the left sounds weird - and the behavior of the first filter `offset` was also a surprise.
Initially, my assumption was that any of these filters would give the same effect to the Sprite, even if the Sprite is moved on the screen.

Let me know!

**Comment by ivanpopelyshev** [comment]
Nope, it means "center of filterArea" and filterArea depends on autoFit and many other things. Yes, this thing is wrong and we have to use different coordinates.

So, are you the hero who can fix all those filters according to my conversion functions? https://github.com/pixijs/pixijs/wiki/v5-Creating-filters#conversion-functions 

btw, if you want those things to be applied on single sprite, its better to do it in mesh-shader: https://pixijs.io/examples/#/mesh-and-shaders/shadertoy-mesh.js
